Tribute-page CSS code

```diff
@@ -0,0 +1,21 @@
+#main {
```

Comment:
noice.. try with classes instead of ids so you can reuse some css

Reply:
yeah, the project had to be done with 'id' specifically to pass it. had like 10 tests to pass